node-tests/e2e/test-support-test.js

```diff
@@ -0,0 +1,112 @@
+const expect = require('chai').expect;
+const AddonTestApp = require('ember-cli-addon-tests').AddonTestApp;
+const fs = require('fs-extra');
+const {
+  removeConfig,
+  setConfig
+} = require('../utils');
+
+describe('e2e: provides test support', function() {
+  this.timeout(300000);
+
+  let app;
+
+  // tests not altering package.json
+  describe('', function() {
+    before(async function() {
+      app = new AddonTestApp();
+
+      await app.create('default', { noFixtures: true });
+
+      // create a simple rendering tests that violates default CSP by using
+      // inline JavaScript
+      let testFolder = 'tests/integration/components';
+      let testFile = `${testFolder}/my-component-test.js`;
+      await fs.ensureDir(app.filePath(testFolder));
+      await fs.writeFile(
+        app.filePath(testFile),
+        "                                                                       \
```

Comment:
I think we can use template strings here, which would make this a bit easier to author and read...

Reply:
Need to escape the backticks used by `htmlbars-inline-precompile` than. So doesn't make a big difference. But refactored to use backticks cause I don't care to be honest. :smile: